test/webhost/TestGameInfo.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import Utils
+import os
+from worlds.AutoWorld import AutoWorldRegister
+
+
+class TestTutorial(unittest.TestCase):
+    def testHasGameInfo(self):
+        import WebHost
+        WebHost.create_ordered_tutorials_file()
+        for game_name, world_type in AutoWorldRegister.world_types.items():
+            if not world_type.hidden:
+                target_path = Utils.local_path("WebHostLib", "static", "generated", "docs", game_name)
+                for game_info_lang in world_type.web.game_info_languages:
+                    with self.subTest(game_name):
+                        try:
```

Comment:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue
assertTrue has a msg=None kwarg, so you don't need the try/except wrapping.